src/polyfem/assembler/RhsAssembler.cpp

```diff
@@ -144,7 +144,7 @@ namespace polyfem
 					{
 						const auto &b = bs.bases[i];
 						const auto &glob = b.global();
-						assert(glob.size() == 1);
+						// assert(glob.size() == 1);
```

Comment:
I had to comment this assertion out to make the NCMesh tests pass. Why did we need this? Maybe it was supposed to be `>= 1`?

Reply:
@Huangzizhou

Reply:
This assertion may fail only with ncmesh, boundary nodes in cmesh only have 1 glob, but boundary nodes in ncmesh may have multiple. I think it's safe to comment it.